cmd/blob-server/main.go

```diff
@@ -121,7 +123,7 @@ func main() {
 		c.JSON(http.StatusOK, job.Status)
 	})
 
-	router.GET("/:daName/verify/:txHash", func(c *gin.Context) {
```

Comment:
would prefer this generic endpoint only. For avail we can treat `txHash` as `${blockHeight}-${extIdx}` and the param basically can be decode in the single `verify module function, which can internally import all verifier modules and choose as per arguement

Reply:
Also, just like we do for posting, we can have VerifierBuilder, which keeps verifier instances and takes jobs for them.

Reply:
Yeah we can keep `/:daName/verify` but will add `txHash`, `height`  as query parameters so one can pass the appropriate params according to the DA.
I think we should keep jobs for later because it does not take much time to verify availability, plus we have not finalized how we're going to use it in `blob-daddy`

Reply:
Looks like this: https://bold-escape-336243.postman.co/workspace/Stackr-Labs~794fc07d-d3f9-42f5-8511-d02508a0161e/request/31704078-4c9f337c-ef02-429c-b7d5-4f99385fa001?action=share&creator=34601608&ctx=documentation

Reply:
add mapping in json/toml/native